### PR TITLE
systemd-tmpfiles: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.es/linux/systemd-tmpfiles.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# systemd-tmpfiles
+
+> Crea, elimina y limpia archivos y directorios volátiles y temporales.
+> Este comando es invocado automáticamente en el arranque por los servicios systemd, en tanto ejecutarlo manualmente no suele ser necesario.
```
Comment (tricantivu) [inline pages.es/linux/systemd-tmpfiles.md:4]:
```suggestion
> Este comando es invocado automáticamente en el arranque por los servicios de systemd, ejecutarlo manualmente tiende a ser innecesario.
```

File: pages.es/linux/systemd-tmpfiles.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Limpia archivos y directorios con los parámetros de edad configurados:
+
+`systemd-tmpfiles --clean`
+
+- Elimina archivos y directorios según lo especificado en la configuración:
+
+`systemd-tmpfiles --remove`
+
+- Aplica operaciones para configuraciones específicas de usuario:
```
Comment (tricantivu) [inline pages.es/linux/systemd-tmpfiles.md:19]:
```suggestion
- Aplica operaciones en archivos de configuración específicos de usuario:
```

File: pages.es/linux/systemd-tmpfiles.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Crea los archivos y directorios especificados en la configuración:
+
+`systemd-tmpfiles --create`
+
+- Limpia archivos y directorios con los parámetros de edad configurados:
+
+`systemd-tmpfiles --clean`
+
+- Elimina archivos y directorios según lo especificado en la configuración:
```
Comment (tricantivu) [inline pages.es/linux/systemd-tmpfiles.md:15]:
```suggestion
- Elimina archivos y directorios según lo especificado en el archivo de configuración:
```

File: pages.es/linux/systemd-tmpfiles.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# systemd-tmpfiles
+
+> Crea, elimina y limpia archivos y directorios volátiles y temporales.
+> Este comando es invocado automáticamente en el arranque por los servicios systemd, en tanto ejecutarlo manualmente no suele ser necesario.
+> Más información: <https://www.freedesktop.org/software/systemd/man/systemd-tmpfiles.html>.
+
+- Crea los archivos y directorios especificados en la configuración:
```
Comment (tricantivu) [inline pages.es/linux/systemd-tmpfiles.md:7]:
```suggestion
- Crea los archivos y directorios especificados en el archivo de configuración:
```

File: pages.es/linux/systemd-tmpfiles.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Elimina archivos y directorios según lo especificado en la configuración:
+
+`systemd-tmpfiles --remove`
+
+- Aplica operaciones para configuraciones específicas de usuario:
+
+`systemd-tmpfiles --create --user`
+
+- Ejecuta las líneas marcadas para el arranque anticipado:
```
Comment (tricantivu) [inline pages.es/linux/systemd-tmpfiles.md:23]:
```suggestion
- Ejecuta las líneas marcadas para el inicio del arranque:
```

I am not 100% percent sure of  the translation from `early` to `inicio`, what do you think @kant?

Comment (kant) [reply pages.es/linux/systemd-tmpfiles.md:23]: Maybe "Ejecuta las líneas marcadas para el arranque inicial"

Comment (tricantivu) [reply pages.es/linux/systemd-tmpfiles.md:23]: But by early I think the original author meant to say at the first moments of the boot process not the first boot.

I will search and ask the author.

Comment (tricantivu) [reply pages.es/linux/systemd-tmpfiles.md:23]: Hello @quantumflo! We are translating your page to Spanish and were wondering what you meant by ``...early boot"?

Comment (kbdharun) [reply pages.es/linux/systemd-tmpfiles.md:23]: This is what I found in the man page locally:

```roff
--create
If this option is passed, all files and directories marked with f,
F, w, d, D, v, p, L, c, b, m in the configuration files are created
or written to. Files and directories marked with z, Z, t, T, a, and
A have their ownership, access mode and security labels set.

...

--boot
Also execute lines with an exclamation mark. Lines that are not
safe to be executed on a running system may be marked in this way.
systemd-tmpfiles is executed in early boot with --boot specified
and will execute those lines. When invoked again later, it should
be called without --boot.
```

IG this command marks the file to be executed in the early/first boot (the first process) after the system starts.

Comment (tricantivu) [reply pages.es/linux/systemd-tmpfiles.md:23]: I think `al inicio` fits for now given the information from the man page, is it okay if I apply the suggestion?

Comment (kant) [reply pages.es/linux/systemd-tmpfiles.md:23]: ok!

**Review by tricantivu — APPROVED** [review]

**Review by kant — COMMENTED** [review]
LGTM

**Pull request merged**